src/main/java/org/gridsuite/voltageinit/server/service/VoltageInitRunContext.java

```diff
@@ -6,17 +6,21 @@
  */
 package org.gridsuite.voltageinit.server.service;
 
-import lombok.Getter;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.ReporterModel;
+import lombok.Data;
 
+import java.util.HashMap;
 import java.util.Map;
 import java.util.Objects;
 import java.util.UUID;
 
 /**
  * @author Etienne Homer <etienne.homer at rte-france.com>
  */
-@Getter
+@Data
```

Comment:
@Getter is fine here. Do not change for @Data

Reply:
but need toString when dumping the object

Reply:
And when do you dump it ?